airsonic-main/src/main/java/org/airsonic/player/service/JWTSecurityService.java

```diff
@@ -25,6 +28,7 @@
     public static final int DEFAULT_DAYS_VALID_FOR = 7;
     private static SecureRandom secureRandom = new SecureRandom();
 
+    @Autowired
     private final SettingsService settingsService;
 
```

Comment:
Let's not annotate any final fields as `@Autowired`. This is a newer class where we were using `contructor-arg` so the contructor should be annotated as `@Autowired` not the fields. The spring folks recommend using autowired constructors over autowired fields, so we should move to do that as much as possible.